pkg/operator/ceph/csi/spec.go

```diff
@@ -391,14 +369,6 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 			}
 			rbdService.Namespace = r.opConfig.OperatorNamespace
 		}
-		enabledDrivers = append(enabledDrivers, driverDetails{
-			name:           RBDDriverShortName,
-			fullName:       RBDDriverName,
-			holderTemplate: RBDPluginHolderTemplatePath,
-			nodeAffinity:   rbdPluginNodeAffinityEnv,
-			toleration:     rbdPluginTolerationsEnv,
-			resource:       rbdPluginResource,
-		})
```

Comment:
It seems like the only code that used this `enabledDrivers` list was related to starting the holder pods. @Madhu-1 please check my work here 🙏

Reply:
Yes thats correct, it looks like to remove it.